kernel/sorts.mli

```diff
@@ -142,6 +142,7 @@ val set  : t
 val prop : t
 val type1  : t
 val qsort : QVar.t -> Univ.Universe.t -> t
+val make : Quality.t -> Univ.Universe.t -> t
```

Comment:
Maybe adding a bit of documentation explaining the role of `qsort` and `make`?

Reply:
For this kind of trivial wrapper I don't think this is required.